lsp-bridge.el

```diff
@@ -323,6 +323,11 @@ Then LSP-Bridge will start by gdb, please send new issue with `*lsp-bridge*' buf
   "Default LSP server for C language, you can choose `clangd' or `ccls'."
   :type 'string)
 
+(defcustom lsp-bridge-php-lsp-server "intelephense"
+  "Default LSP server for PHP language, you can choose `intelephense' or `phpactor'."
+  :type 'string
+  :safe #'stringp)
```

Comment:
I want to switch servers in `.dir-locals.el` per project in PHP, so I mark it as a [safe variable](https://www.gnu.org/software/emacs/manual/html_node/emacs/Safe-File-Variables.html) by setting `:safe`.